client/src/components/EditProfile/Payment/Payment.tsx

```diff
@@ -0,0 +1,23 @@
+import { Elements } from '@stripe/react-stripe-js';
+import { loadStripe } from '@stripe/stripe-js';
+import CheckoutForm from './PaymentForm/CheckoutForm';
+
+const PUBLISHABLE_KEY =
```

Comment:
I don't think we need to store empty strings as keys in the absence of environment variables :)

Reply:
Let's use the payment profile instead of one-time payment method. May I remove the whole component here?

Reply:
I think you can leave it in for now 👍🏽